umbrui.py

```diff
@@ -27,48 +30,51 @@ def __init__(self):
         self.init_screen()
         
         # Init GRPC connections
+        self.btc_grpc = BtcGRPC()
```

Comment:
This is correct here, because GRPC is a version of RPC

```suggestion
        # Init RPC connections
        self.btc_grpc = BtcRPC()
```